libs/features/commerce/src/ims.js

```diff
@@ -0,0 +1,32 @@
+// @ts-nocheck
+export const IMS_POLL_INTERVAL = 200;
+export const IMS_POLL_MAX_ATTEMPTS = 25;
+
+/** @type {Commerce.pollImsCountry} */
+export function pollImsCountry({
```

Comment:
it is time to check whether gnav can provide a better implementation to this?

Reply:
added todo comment in the code